containers/kubernetes/api-cli/managing-storage.mdx

```diff
@@ -18,175 +18,69 @@ categories:
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
   - You have [created](/containers/kubernetes/how-to/create-cluster/) a Kubernetes cluster running on Scaleway Instances (v1.21+)
-  - You have a Scaleway Project or Organization ID, [Access and Secret key](/identity-and-access-management/iam/how-to/create-api-keys/)
 </Message>
 
+The Scaleway Block Volume CSI driver is responsible for the management of Scaleway Block Volumes within Kubernetes. Pre-installed on Kubernetes Kapsule and Kosmos clusters, this driver allows you to control Block Volumes directly from your orchestration environment. The plugin resides in the `kube-system` namespace within the pods.
 
-The [Scaleway Block Volume](https://www.scaleway.com/en/block-storage/) Container Storage Interface (CSI) driver is an implementation of the [CSI interface](https://github.com/container-storage-interface/spec/blob/master/spec.md) to provide a way to manage Scaleway Block Volumes through a container orchestration system, like Kubernetes. It is installed by default on every Kubernetes Kapsule and Kosmos cluster.
-
-
-## Features
-
-Following is a list of functionalities implemented by the Scaleway CSI driver.
-
-### Block device resizing
-
-The Scaleway CSI driver implements the resize feature ([example for Kubernetes](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/)). It allows an online resize (without the need to detach the block device). However, resizing can only be done upwards, decreasing a volume's size is not supported.
+<Message type="tip">
+    Refer to our video tutorial [Getting Started with Kubernetes Part 4 - Storage](/containers/kubernetes/videos/) to view a visual presentation and step-by-step guidance of how to manage Block Storage volumes on Kubernetes with the Scaleway CSI.
+</Message>
```

Comment:
This isn't published yet FYI! Should be on or after 1st Feb though, if no delays